index.js

```diff
@@ -1,9 +0,0 @@
-/**
- * @format
- */
-
-import { AppRegistry } from 'react-native';
-import { name as appName } from './Example/app.json';
-import App from './Example/src/App';
-
-AppRegistry.registerComponent(appName, () => App);
```

Comment:
😅

Reply:
btw, do you know what the heck was it? :D

Reply:
I guess you've used it in order to run RN when "main" property of package.json was equal to "index.js" =)

Reply:
I hope I wasn't drunk when I did it, it just doesn't make sense for me now :D